pkg/controller/replicator.go

```diff
@@ -292,6 +292,10 @@ func (r *replicator) Close() error {
 		if backend.mode == types.ERR {
 			continue
 		}
+
+		// Stop the monitoring goroutine in the Controller
```

Comment:
The code is pretty obvious by itself, so don't need the comment.

Reply:
Removed.